parasolr/tests/test_query.py

```diff
@@ -90,6 +91,60 @@ def test_count(self):
         # cache should not be populated
         assert not sqs._result_cache
 
+    def test_facet_fields(self):
+        mocksolr = Mock(spec=SolrClient)
```

Comment:
I'm not sure this test is actually testing the right things - it should be completely agnostic as to the specifics of the Solr response. Maybe better to patch `QueryResponse` and check that it is called or not called as expected? Check that you get the right return type, not the specific values.